src/components/content/heroes/weather-api.ts

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+const API_KEY = process.env.WEATHER_API_KEY;
+const latitude = 48.1351;
+const longitude = 11.582;
+
+async function getWeather() {
+  const apiUrl = `https://api.weatherapi.com/v1/current.json?key=${API_KEY}&q=${latitude},${longitude}`;
+
+  try {
+    const response = await axios.get(apiUrl);
+    return response.data.current.condition.text;
+  } catch (error) {
+    console.error('Error fetching weather:', error);
+    throw new Error('Unable to fetch weather information.');
```

Comment:
I am not sure if should throw an error here, since the easteregg is no crucial functionality. I would suggest to catch the error and return the default weather state.

Our page should not break if the weather api does not respond for some reason (e.g. no token/invalid token in the environment)

```suggestion
    return "Not Set";
```